### PR TITLE
Add placeholder to search text field

### LINKED ISSUE #550: Add Placeholder Text to Search Field
**Issue: Add Placeholder Text to Search Field**

**Idea Description**
The search field in the app currently lacks a placeholder to guide users. Adding the placeholder text "気になる技術を入力" (Enter a technology of interest) will provide users with a clear hint about what they should input, enhancing user experience.
image
https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI

### PR DESCRIPTION
**Pull request by hiraike32**
## Issue
- close #550 

## Overview (Required)
- Add placeholder to search text field
- Create compose preview for easy checking

## Links
- [Figma](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=55052%3A45676&mode=dev)

## Screenshot
Before | After | After(English)
:--: | :--: | :--:
 |  |

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SessionsStrings.kt [hunk 1]
```diff
@@ -72,6 +74,7 @@ sealed class SessionsStrings : Strings<SessionsStrings>(Bindings) {
                 UserIcon -> "User icon"
                 EventDay -> "Day"
                 Category -> "Category"
+                SearchPlaceHolder -> "Enter some technology"
```
Comment (hiraike32) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SessionsStrings.kt:77]: Placeholder can't display many characters, so I made it simple. Please let me know if any corrections are needed.

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SearchTextFieldAppBar.kt [hunk 2]
```diff
@@ -122,3 +133,13 @@ private fun SearchTextField(
         },
     )
 }
+
+@Preview
+@Composable
+fun SearchTextFieldAppBarPreview() {
```
Comment (hiraike32) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SearchTextFieldAppBar.kt:139]: I made a preview for easy checking.

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SessionsStrings.kt (hunk 1 above)
Comment (takahirom) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SessionsStrings.kt:77]: 👍

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SearchTextFieldAppBar.kt (hunk 2 above)
Comment (takahirom) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SearchTextFieldAppBar.kt:139]: Nice!

**Comment by hiraike32** [comment]
I don't understand why CI is failing. please help me.

**Comment by takahirom** [comment]
I'll check this 🙇

**Comment by takahirom** [comment]
I'm going to create an issue but can you apply this for now? 🙇 

```kotlin
    LaunchedEffect(Unit) {
        if (!isTest()) {
            focusRequester.requestFocus()
        }
    }
```

**Review by takahirom — APPROVED** [review]
Looks great! Sorry for bothering you!

**Pull request merged**